bindata/assets/deployments/console-deployment.yaml

```diff
@@ -70,11 +70,15 @@ spec:
               path: /health
               port: 8443
               scheme: HTTPS
-            initialDelaySeconds: 150
-            timeoutSeconds: 1
+            failureThreshold: 1
             periodSeconds: 10
-            successThreshold: 1
-            failureThreshold: 3
+          startupProbe:
```

Comment:
I like the idea of this change.  This makes the startup wait 5 minutes instead of 2.5 minutes. That seems ok to me if you find it reasonable, but I want to be sure the difference is obvious.

Reply:
Yes, this was intentional. In the worst case, the console will take 5 minutes to start. See https://github.com/openshift/console/pull/13610